scripts/release-update-tilt-repo.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
above comments that are accepted should also be applied to this file

Reply:
done